tests/api/ShowCardEndpointCest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Tests;
+
+use App\Tests\ApiTester;
+use Codeception\Util\HttpCode;
+use App\Entity\Card;
+
+class ShowCardEndpointCest
+{
+    public function _before(ApiTester $I)
+    {
+    }
+
+    public function testItShowsCardDetails(ApiTester $I)
+    {
+        $cardid = $I->haveInDatabase('Card', ['name' => 'GoblinTest', 'description' => 'Low lvl', 'attack' => 1, 'defense' => 1, 'hp' => 1]);
```

Comment:
Its minor for now but if we have array which is that 'big' then we could format it line by line:
```
$I->haveInDatabase(
    'Card',
    [
        'name' => 'xxx',
        'xxx' => 123,
        ...
     ]
); 
```
usually this approach make it more readable.

Reply:
done